firmware/common2015/utils/robot-types.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "RTP.hpp"
+
```

Comment:
We should get rid of the defines for NULL, TRUE, and FALSE here - nullptr, true, and false are all part of the c++ standard.